invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/header.overrides

```diff
@@ -39,3 +39,7 @@ aside.sidebar {
     display: none;
   }
 }
+
+.ui.header  {
```

Comment:
this is modifyng the theme globally. Are you sure this is what you want to do?

Reply:
Are there any downsides of having it apply to all? It will only make sure that no headings are overflowing its container, no?

Reply:
Maybe it could create an oversized container that causes layout problems. A better solution would be to truncate IMO, but as a default case I think that word-break: break-word seems better.